lib/components/util/dropdown.tsx

```diff
@@ -192,8 +195,25 @@ const Dropdown = ({
           {children}
         </DropdownMenu>
       )}
-    </DropdownContainer>
+    </DropdownContainerEl>
   )
 }
 
-export default Dropdown
+export const SortResultsDropdown = styled(Dropdown)`
+    position: relative;
+
+    ${DropdownButton} {
+      border-radius: 5px;
+      padding: 3px 7px;
+      min-width: 130px;
```

Comment:
What is the intended effect for min-width?

Reply:
To keep the button size consistent instead of resizing according to contents. Although I do agree for options such as "cost" there's a lot of padding. I can remove both the `min-width` and the `text-align`!